src/login/Login.jsx

```diff
@@ -92,7 +92,13 @@ export default function Login() {
                         </div>
                         <div className="row button-group">
                             <div className="col">
-                                    <a className="btn btn-primary" id="registerbtn" href="/register" role="button">Sign Up</a>
+                                <Link 
+                                    to="/register"
```

Comment:
please remove unnecessary changes.

Reply:
Thanks for review this @nlok5923 , but I think it is better to use Link instead of anchor tags to navigate to some other route, what's your view @mtreacy002 ma'am, should I revert this change ?

Reply:
@sachinsom93 @nlok5923  It is better to use `Link` in `react` as anchor tags trigger a page refresh which would reset the application states whereas the `Link` and `Navlink` of react-router doesn't trigger a page refresh. In single page apps with `react`, it is better to use `Link` from `react-router`.

Reply:
@nlok5923 Just as an extra opinion, I agree with @sachinsom93 and @Rahulm2310 on the use of `Link` tags in place of `a` in React for the same reasons specified above. Generally speaking, the `Link` tags are better placed for in-app navigation(relative urls) and the `a` tags for external links(absolute urls).